about.html

```diff
@@ -0,0 +1,258 @@
+                  />
+                </a>
+              </li>
+              <li class="language-container item-size">
+                <a href="#" class="language">
+                  <span class="language-name text-color">English</span>
+                </a>
+              </li>
+              <li class="my-page-container item-size">
+                <a href="#" class="mypage">
+                  <span class="text-color">My Page</span>
+                </a>
+              </li>
+              <li class="logout-container item-size">
+                <span class="logout text-color">Logout</span>
+              </li>
+            </ul>
+          </div>
+          <div class="top-sub-navbar-2">
+            <div class="top-sub-navbar-2-logo-container">
+              <a href="index.html">
+                <img
+                  src="images/Opolo-global-logo.png"
+                  alt="logo icon"
+                  class="top-sub-navbar-2-logo"
+                />
+              </a>
+            </div>
+            <ul class="top-sub-navbar-2-items-container">
+              <li class="about-container item-style">
+                <a href="#" class="about text-color-2">About</a>
+              </li>
+              <li class="program-container item-style">
+                <a href="#" class="program text-color-2">Program</a>
+              </li>
+              <li class="join-container item-style">
+                <a href="" class="join text-color-2">Join</a>
+              </li>
+              <li class="sponser-container item-style">
+                <a href="#" class="sponser text-color-2">Sponser</a>
+              </li>
+              <li class="news-container item-style">
+                <a href="#" class="news text-color-2">News</a>
+              </li>
+              <li class="cc-compaign-cantainer item-style">
+                <a href="#" class="cc-campaign text-color-2">Campaign</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+        <nav class="nav-container">
+          <div class="hamburger-icon-container">
+            <img
+              src="images/mobile_menu.png"
+              alt="hamburger-icon"
+              class="hamburger-icon"
+            />
+          </div>
+        </nav>
+        <div class="home-page-title-container">
+          <h2 class="home-page-primary-text">"Hello Developers"</h2>
+          <h1 class="home-page-title-text">NEW CLASSES START DECEMBER 2022</h1>
+        </div>
+        <div class="home-page-supporting-text-model">
+          <p class="home-page-supporting-text">
+            Opolo Global Web Developers submit a meeting place  where you will become a professional web developer. 
+            We know the demand of the market and provide the best materials for our students. Global Web Developers is a 
+            project-based course that gives you a job.
+          </p>
+        </div>
+        <div class="contact-container">
```

Comment:
Same here, as the [last code reviewer suggested](https://github.com/EngruugbeJimmy/Opolo_Conference_Webpage/pull/1#discussion_r1007987986) kindly please remove the white spacing on the right side of your project. :+1: :smiley: :smiley: Also please make sure that the text is not overlapping the box border.
![image](https://user-images.githubusercontent.com/98363075/198755452-61a8a494-3786-4269-8572-ebfdb7f2936d.png)